### PR TITLE
[chore] Correct the java doc in funtion DagHelperTest generateDag2

### PR DESCRIPTION
**Pull request by yimaixinchen**
adding more details in the dagHelperTest.
adding the dag graph can understand dag easily.

### CODE REVIEW

File: dolphinscheduler-dao/src/test/java/org/apache/dolphinscheduler/dao/utils/DagHelperTest.java [hunk 1]
```diff
@@ -280,9 +280,15 @@ public void testConditionPostNode() throws IOException {
...
+     * process:
      * 1->2->3->5->7
      * 4->3->6
-     * 2->8->5->7
+     * 1->2->8->5->7
+     * DAG graph:
+     *    8->5->7
+     *    ↑  ↑
+     * 1->2->3->6
+     *    4-↗
```
Comment (zhongjiajie) [inline dolphinscheduler-dao/src/test/java/org/apache/dolphinscheduler/dao/utils/DagHelperTest.java]: I have an experience in Python api about draw a graph about DAG, and maybe we should keep the longest flow into online `1-2-3-5-7` and the finial is like

```suggestion
     *      4 ->   -> 6
     *          \ /
     * 1 -> 2 -> 3 -> 5 -> 7
     *       \       /
     *        -> 8 ->
```

Comment (zhongjiajie) [inline dolphinscheduler-dao/src/test/java/org/apache/dolphinscheduler/dao/utils/DagHelperTest.java]: I have an experience in Python api about draw a graph about DAG, and maybe we should keep the longest flow into online `1-2-3-5-7` and the finial is like

```suggestion
     *      4 ->   -> 6
     *          \ /
     * 1 -> 2 -> 3 -> 5 -> 7
     *       \       /
     *        -> 8 ->
```

**Comment by zhongjiajie** [comment]
@yimaixinchen Hi, I give some suggestion on your PR

**Comment by yimaixinchen** [comment]
@zhongjiajie  thank you for your suggestion.keeping the longest flow is necessory.I update the commit.

**Review by zhongjiajie — APPROVED** [review]
LGTM, thanks

**Pull request merged**